src/store/index.js

```diff
@@ -174,7 +174,7 @@ const initializeAppLocal = (commit, dispatch, state) => {
         .then(appInfo => {
           clearStateIfDnaChanged(appInfo, commit, dispatch, state);
 
-          const cellId = appInfo.cell_data[0][0];
+          const cellId = appInfo.cell_data[0].cell_id;
```

Comment:
Thanks for pointing this out!  It might be small, but it's def an important change (...and one that we'll actually need to repeat across a few different repos in holo).